core/modules/system/system.module

```diff
@@ -4009,7 +4009,7 @@ function system_settings_form($form) {
   $form['actions']['submit'] = array('#type' => 'submit', '#value' => t('Save configuration'));
 
   if (empty($form['#config']) && !settings_get('backdrop_drupal_compatibility')) {
-    drupal_set_message(t('Drupal compatibility layer is disabled and this form has not be updated to use Backdrop configuration management.  Changes will not be saved.'), 'warning');
+    backdrop_set_message(t('Drupal compatibility layer is disabled and this form has not be updated to use Backdrop configuration management.  Changes will not be saved.'), 'warning');
```

Comment:
nits:
- grammar: "has not be**en** updated"
- double space: `...management.  Changes...`